rep-0146.rst

```diff
@@ -0,0 +1,80 @@
+REP: 146
+Title: ROS Robot Parameter Naming Convention
+Author: Kei Okada <k-okada@jsk.t.u-tokyo.ac.jp>
+Status: Draft
+Type: Informational
+Content-Type: text/x-rst
+Created: 31-Jly-2015
+Post-History:
+
+Abstract
+========
+
+This REP aims to define the best practices for robot related parameter
+name space for making robot-agnostic packages.
+
+
+Motivation
+==========
+
+As the number of ROS-supported robot increases, we will write
+robot-agnostic packages, but still they need robot-specific
+information as calibration parameters.
+
+A `robot_description` parameter is widely shared naming convention
+among ROS community and this REP proposes such conventions for robot specific information on
+parameter, so application program can build their program on top of that.
+
+
+Specifications
+==============
+
+Parameters
+----------
+
+robot_description
+'''''''''''''''''
+
+`robot_description` stores robot kinematics / geometry information using
+URDF format.
+
+
+robot/type
+''''''''''
+
+`robot/type` stores robot type, for example 'pr2' in PR2 robot case [1]
```

Comment:
It would be good to define the type more clearly. One example is not enough to clearly define what should be here.

Reply:
add Pepper and Fetch case for `robot_type`
I also added example of `robot_name` if you know other PR2 name, please let me know 
% Unfortunately, we JSK calls them pr2, or pr1012...